GNUmakefile

```diff
@@ -22,10 +22,15 @@
 
 SRC:=                      \
        black-scholes.c     \
-       black-scholes_mkl.c \
        data_gen.c          \
        main.c
 
+MKL ?= 1
+
+ifeq ($(MKL),1)
+	SRC += black-scholes_mkl.c
+endif
+
```

Comment:
BTW, why not to create a separate target instead?

Reply:
done!